lib/browser/rpc.js

```diff
@@ -88,38 +88,37 @@ export function asyncOpenRealm(id, config, callback) {
     }));
 }
 
-export function _anonymousRPC(args) {
-    args = args.map((arg) => serialize(null, arg));
-    const result = sendRequest('_anonymous', { arguments: args });
+export function _anonymousRPC() {
+    const result = sendRequest('_anonymous', { arguments: undefined });
     return deserialize(undefined, result);
 }
 
-export function _appleRPC(args) {
-    args = args.map((arg) => serialize(null, arg));
+export function _appleRPC(token) {
+    const args = Array.prototype.map.call(arguments, (arg) => serialize(null, arg));
```

Comment:
Could also be achieved with:
```js
export function _appleRPC(...args) {
    const result = sendRequest('_apple', {
        arguments: args.map((arg) => serialize(null, arg))
    });
    return deserialize(undefined, result);
```
The arguments are already named in the public methods.

Reply:
thanks, but I don't like this. any method regardless if public or not should name the arguments. Maybe I am to protective, but would you do that if this was TypeScript.

Reply:
Yes I would, but also just ES6+. `...args` declares that everything passed to the function is (potentially) used. (In TS you would further declare the types allowed in the args array).
I'm not sure I agree that named arguments is more protective, in this case, as the function disregards them by utilizing `arguments` instead.